apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -11,47 +11,145 @@
+              Profiles View
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <div v-else>
+        <v-list-item title="View profiles">
+          <v-list-item-avatar>
+            <v-icon small>mdi-television-guide</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              Profiles View
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </div>
+      <v-subheader>Files</v-subheader>
+      <v-expansion-panels v-model="file_views" flat="true">
+        <v-expansion-panel>
+          <div v-if="visible_evaluation_files.length > 0">
+            <v-expansion-panel-header title="Results">
+              <v-list-item>Results</v-list-item>
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <FileLister
+                v-for="(file, i) in visible_evaluation_files"
+                :key="i"
+                :file="file"
+              />
+              <v-list-item
+                title="Toggle selection on all results"
+                @click="toggle_all_evaluations"
+              >
+                <v-list-item-avatar>
+                  <v-icon small>mdi-format-list-bulleted</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <div v-if="all_toggled_evaluation">
+                      Deselect all results set
+                    </div>
+                    <div v-else>Select all results set</div>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </v-expansion-panel-content>
+          </div>
+        </v-expansion-panel>
+        <v-expansion-panel>
+          <div v-if="visible_profile_files.length > 0">
+            <v-expansion-panel-header title="Profiles">
+              <v-list-item>Profiles</v-list-item>
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              <FileLister
+                v-for="(file, i) in visible_profile_files"
+                :key="i"
+                :file="file"
+              />
+              <v-list-item
+                title="Toggle selection of all profiles"
+                @click="toggle_all_profiles"
+              >
+                <v-list-item-avatar>
+                  <v-icon small>mdi-format-list-bulleted</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <div v-if="all_toggled_profiles">Deselect all profiles</div>
+                    <div v-else>Select all profiles</div>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </v-expansion-panel-content>
+          </div>
+        </v-expansion-panel>
```

Comment:
The `v-expansion-panel` here should be a component, it should be possible to leverage the same component for the `Results` and `Profiles` expansion components.